testcafe/tests/pages/drive/drive-model-private.js

```diff
@@ -1,13 +1,40 @@
 import { t } from 'testcafe'
 import {
+  getElementWithTestId,
   getPageUrl,
   isExistingAndVisibile,
   overwriteCopyCommand,
   getLastExecutedCommand
 } from '../../helpers/utils'
 import DrivePage from './drive-model'
 
-export default class PrivateDrivePage extends DrivePage {
+import Modal from './drive-modal-model'
+const moveModale = new Modal()
```

Comment:
Is this the best place to initialize this? Wouldn't it make more sense to do it in the constructor for example?